src/engine/sparqlExpressions/ConvertToNumericExpression.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2024
```

Comment:
Complete the copyright notice according to the other files (add yourself as the author).